web-ui/src/pages/BirthdayReportPage.jsx

```diff
@@ -55,13 +69,12 @@ const BirthdayReportPage = () => {
           multiple
           id="monthSelect"
           options={months}
-          defaultValue={[months[currentMonth].month]}
+          defaultValue={defaultMonths}
           value={selectedMonths}
           onChange={onMonthChange}
           isOptionEqualToValue={(option, value) => {
-            return value ? value.month === option.month : false;
+            return value ? value === option : false;
           }}
-          getOptionLabel={option => option.month}
```

Comment:
We no longer need the `getOptionLabel` property since the options are already the strings we want.